feature/bgp/routereflector/route-reflector-test

```diff
@@ -0,0 +1,45 @@
+Summary: 
+ - BGP route reflector capabilities check
+ - Ensure functionality of different OC paths for "supported-capabilities", "BGP peer-type", "BGP Neighbor details" and "BGP transport session parameters"
+
+Topology:
+ATE (Port1) <-EBGP-> (Port1) DUT (Port2) <-IS-IS/IBGP-> (Port2) ATE, DUT (Port3) <-IS-IS/IBGP-> (Port3) ATE
+  - Connect ATE Port1 to DUT port1 (EBGP peering)
+  - Connect ATE Port2 to DUT port2 (For IS-IS adjacency and IBGP peer reachaility)
+  - Connect ATE Port3 to DUT port3 (For IS-IS adjacency and IBGP peer reachaility)
+
+Procedure:
+  - Establish IS-IS adjacency between ATE Port2 <-> DUT Port2, ATE Port3 <-> DUT Port3. 
+  - Establish BGP sessions as follows between ATE and DUT . 
+    - ATE port 2 and ATE port3 are emulating RR clients peered with the DUT acting as the RR server. DUT's loopback address should be used for the IBGP peering and 
+      "transport/config/local-address" OC path should be used on DUT to configure BGP transport address for IBGP peering. ATE addresses used for the IBGP peering (different from ATE Port1 and ATE Port2 addreses) and DUT 
+      loopback addresses should be reachable via IS-IS. Each of the IBGP peering should advertise 5M IPv4 and 1M IPv6 prefixes.
+    - The DUT Port1 has eBGP peering with ATE Port 1 and is receiving 10M IPv4 and 3M IPv6 prefixes. Please note: The FIB is expected to be 1.5M v4 routes and 600K v6 routes. 
```

Comment:
Related to my comment above: How do we expect the RIB->FIB math to work out here? If there are 5M unique prefixes each from 2 RRCs, how did we reduce them to 1.5M FIB entries etc? For that matter, is validation of the FIB something that is needed in the scope of the RR test?

Reply:
Well, I thought it is implicit here that 1.5M v4 and 600k v6 are unique, not clear? Didnt mention policies here. These are prefixes learnt from a single EBGP peer simulating an external peer w/ internet scale routes. 

Anyways, I added more text to clarify

Reply:
For the FIB, of the  1.5M v4 routes and 600K v6 routes, how many of those are from eBGP vs. iBGP?   Maybe best to explain how many unique prefixes there are for each iBGP and the one eBGP session.

Reply:
Ok, there is a good explanation for the eBGP session unique prefixes.  But need the same for the iBGP sessions

Reply:
Added more clarity. Hopefully that helps.